bin/i18n/i18n_script_utils.rb

```diff
@@ -142,3 +144,21 @@ def restore(source, redacted, dest, *plugins)
   source_json.close
   redacted_json.close
 end
+
+def get_level_url_key(script, level)
+  script_name = script.name
+  script_level = level.script_levels.find_by_script_id(script.id)
+
+  "https://studio.code.org/s/#{script_name}/stage/#{script_level.stage.relative_position}/puzzle/#{script_level.position}"
+end
+
+def get_level_from_url(url)
```

Comment:
is there a specific reason not to do this with a regex?

Reply:
Changed!